stream.js

```diff
@@ -21,10 +21,10 @@ const STREAM_STATE = Object.freeze({
   ENDED: 4,
 })
 
-function Stream(blocks, opts) {
+function Stream(log, opts) {
   opts = opts || {}
 
-  this.blocks = blocks
+  this.log = log
```

Comment:
It was always confusing to me that `this.blocks.something()` was an API on the log, so this rename helps in that sense.